### DISCUSSION TITLE
How to show listners number on html page ?

### DISCUSSION THREAD

**Discussion by Mozitux** [AzuraCast/AzuraCast#5597] [Q&A] [answered]
Hello,
I'm a bit lost with the Azuracast API, I'm simply trying to display the number of listeners on an HTML page, does anyone have an idea how to proceed?

Thank you :)

**Comment by KazeNoTachi** [comment, marked as answer]
Here is a bit of JS/html code to grab the total number of current listeners from every mount point. 

```
<html>
<head>
<meta charset="utf-8">
<title>Untitled Document</title>
<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
</head> 
<body>
    <h1 id="listeners"></h1>
    <script>
        var json = $.getJSON("https://demo.azuracast.com/api/station/1", function(data) {
        //take  $..mounts[*]..current and sum the value from json
        console.log(data)
        var sum = 0;
        $.each(data.mounts, function(i, item) {
            sum += item.listeners.current;
            document.getElementById("listeners").innerHTML = "Current Listeners: " + sum;
    
        });
        
    });
        </script>
</body>
</html>
```

**Reply by Mozitux** [reply]
Thanks, it's work fine!

**Reply by ayprof2020** [reply]
How im I doing to use it on my radio

**Reply by Mozitux** [reply]
> How im I doing to use it on my radio

Simply paste it in an .html file and insert the HTML page in an iframe.

_**EDIT:** Of course you will have to adapt the content to your station for it to work._

**Reply by ayprof2020** [reply]
Please can you do a YouTube tutorial or cast your screen on how to do it.
